### PR TITLE
test: Pass KERNEL env param in ginkgo-kernel job

### PR DESCRIPTION
**Pull request by brb**
Otherwise, the `RunsOnNetNextOr419Kernel()` returns `false` on the 4.19 build, which makes to skip some kube-proxy replacement related tests.

Also, disable the dynamic kernel vsn selection, as we currently run the job only on the 4.19 kernel.

Fixes: 72fe87afb80a ("test: Extend RunsOnNetNext helper family")

### CODE REVIEW

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-me-please

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

File: jenkinsfiles/ginkgo-kernel.Jenkinsfile [hunk 1]
```diff
@@ -175,3 +173,10 @@ pipeline {
         }
     }
 }
+
+def get_kernel(String default_kernel) {
+    dir("${TESTDIR}") {
```
Comment (nebril) [inline jenkinsfiles/ginkgo-kernel.Jenkinsfile]: Seems like this line causes the problem in CI.

File: jenkinsfiles/ginkgo-kernel.Jenkinsfile [hunk 2]
```diff
@@ -12,6 +12,9 @@ pipeline {
         TESTED_SUITE="k8s-${K8S_VERSION}"
         GINKGO_TIMEOUT="300m"
         DEFAULT_KERNEL="419"
+        KERNEL = get_kernel("${DEFAULT_KERNEL}")
```
Comment (nebril) [inline jenkinsfiles/ginkgo-kernel.Jenkinsfile]: If you set `KERNEL` here in a manner similar to https://github.com/cilium/cilium/blob/master/jenkinsfiles/ginkgo-gke.Jenkinsfile#L128 , it should be available as an env var in all children pipeline steps.

**Review by nebril — CHANGES_REQUESTED** [review]

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by borkmann** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
test-with-kernel

**Comment by brb** [comment]
Closing in favor of https://github.com/cilium/cilium/pull/11915 and https://github.com/cilium/cilium/pull/11894.

**Pull request closed without merging**